### ISSUE TITLE
Wrong version number in MacOS download (9.3.0 instead of 9.3.7)

### ISSUE THREAD

**Issue by suschizu** [IBM/kui#6499]
**Describe the bug**
The macOS download contains 9.3.0 instead of 9.3.7

```
info.plist
    <key>CFBundleShortVersionString</key>
    <string>9.3.0</string>
    <key>CFBundleVersion</key>
    <string>9.3.0</string>

```

**Screenshots**

<img width="359" alt="Screen Shot 2021-01-06 at 20 24 11" src="https://user-images.githubusercontent.com/16004333/103811886-eb9c4600-505d-11eb-9abd-1646c5604c35.png">

**Code snippets**
If applicable, add code samples to help explain your problem.

**System**

- Kui version:  9.3.0 (should be 9.3.7) 
- Operating system: macos
-

**Comment by starpit** [comment]
Thanks for the bug report. Good catch! This looks like a bug in the way we transfer the current version to electron-packager. The `version` command in Kui looks good, but I can confirm that indeed the platform binary metadata seems incorrect (at least for macOS).

**Comment by myan9** [comment]
Fixed by https://github.com/IBM/kui/pull/6508

**Comment by suschizu** [comment]
Great thanks :)

**Comment by starpit** [comment]
This fix will be released in 9.3.8. Thanks for the help with debugging Kui!
